docs/addons/configure-addons.md

```diff
@@ -0,0 +1,40 @@
+---
+title: 'Configure and communicate with an addon'
+---
+
+Storybook is designed to be highly customizable. You can develop all kinds of workflows around it. To support this, you can allow users to configure and communicate with addons in several different ways.
```

Comment:
Clarify who this article is for. 

Who are "users" in this case? Addon authors or end users? If the former the parameters section might need to be reframed. If the latter, the preset and channel sections might need to be reframed.

Reply:
@domyen the audience is meant to be Addon authors. I updated the intro to clarify this. The rest of the sections work as is.